app/Models/Entry.php

```diff
@@ -226,7 +226,7 @@ public function content(bool $withEnclosures = true, bool $allowDuplicateEnclosu
 			$description = nl2br($enclosure['description'] ?? '', true);
 			$length = $enclosure['length'] ?? 0;
 			$medium = $enclosure['medium'] ?? '';
-			$mime = $enclosure['type'] ?? '';
+			$mime = ($enclosure['type'] ?? '');
```

Comment:
Any reason for this change?

Reply:
Parenthesis is useless

Reply:
Indeed, so why did you **add** the parentheses?

Reply:
Oups ! Good review